client/register/register.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+<h3>Cadastro</h3>
+<form *ngIf="active" #f="ngForm" novalidate (ngSubmit)="onSubmit()">
```

Comment:
Formulário é exibido apenas se `RegisterComponent.active` for `true`. O nome do formulário é `f` e ao submeter, ele executa a função `onSubmit`.